openmetadata-ui/src/main/resources/ui/src/styles/temp.css

```diff
@@ -814,6 +814,9 @@ body .list-option.rdw-option-active {
 
 :root {
   --ant-navbar-height: 64px;
+  &::-webkit-scrollbar {
+    display: none;
+  }
```

Comment:
@aniketkatkar97 this will hide scrollbars fully, We want to adjust the layout if the scrollbar is visible. 
This will help user to understand if there's a scroll or not